core/typed-events/skills/react-typed-events/SKILL.md

```
@@ -0,0 +1,142 @@
+type: framework
+library: '@stephansama/typed-events'
+framework: react
+library_version: "3.0.9"
+requires:
+  - typed-events
+sources:
+  - stephansama/packages:core/typed-events/src/react.ts
+  - stephansama/packages:core/typed-events/README.md
+---
+
+This skill builds on `typed-events`. Read it first for foundational concepts including factory selection, cleanup patterns, and SSR behaviour.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)